packages/techdocs-cli/src/commands/index.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CommanderStatic } from "commander";
+
+export function registerCommands(program: CommanderStatic) {
+  program
+    .command("generate")
+    .description("Generate TechDocs documentation site using mkdocs.")
+    .option(
+      "--source-dir <PATH>",
+      "Source directory containing mkdocs.yml and docs/ directory.",
+      "."
+    )
+    .option(
+      "--output-dir <PATH>",
+      "Output directory containing generated TechDocs site.",
+      "./site/"
+    )
+    .option(
+      "--no-docker",
+      "Do not use docker, use mkdocs executable and plugins in current user environment."
```

Comment:
```suggestion
      "Do not use Docker, use MkDocs executable and plugins in current user environment."
```

Nit to use product's std capitalization.